### ISSUE TITLE
nuplan v1.1 dataset 

### ISSUE THREAD

**Issue by chao-SFD** [motional/nuplan-devkit#175]
The downloaded file from the newest dataset（nuplan-v1.1） seems to be the old one. The Md5 of nuplan Map in the version v1.1 is the same as the version v1.1, which is not as the page described.

**Comment by patk-motional** [comment]
Hi @chao-SFD,

Which files exactly did you download?

As for the map, the v1.0 and v.1.1 maps should be the same. Let me see why the md5 is different

**Comment by chao-SFD** [comment]
Yes，I downloaded the map file in  V1.1 dataset.  And the md5 is not as described on the download page，but has the same value compared with the map file in  V1.0 dataset.

> Hi @chao-SFD,
> 
> Which files exactly did you download?
> 
> As for the map, the v1.0 and v.1.1 maps should be the same. Let me see why the md5 is different

**Comment by patk-motional** [comment]
So you are only talking about the map, not the db files?

**Comment by chao-SFD** [comment]
> So you are only talking about the map, not the db files?

Yes, I have downloaded the v1.1_mini.zip, the db file seems to be correct. And the downloaded map file name in v1.1 dataset  is still "nuplan-maps-v1.0“. So I dont know if the map file should be same or not between v1.0 and v1.1.

**Comment by patk-motional** [comment]
They should be functionally the same. We uploaded it for completeness' sake. Let me double check if there are any differences that you should be aware of

**Comment by Jeff09** [comment]
Hi @patk-motional ,

As for the db files, the V1.0 got 72 db files in the mini-dataset, while the V1.1 had only 64 db files in the same mini-dataset. And also the lidar-pc files in the V1.1 mini-dataset is almost half as that in the V1.0. I wonder what has changed and can I still use V1.0 dataset when using nuplan-devkit 1.0 version code? Thanks.

**Comment by patk-motional** [comment]
@chao-SFD,

Apologies for the late reply. The map hash should both be `71fef4933dba53125d7cee8b1e7e9ca5`. We'll update the hash on the website  to reflect this.

**Comment by patk-motional** [comment]
@Jeff09,

We had to filter out a lot of poor data from v1.0.

The short answer is the dataset is backward compatible. However, the data quality and some data annotation may be different between the two dataset versions.
